test/DividendPayableToken.dividend.test.js

```diff
@@ -80,7 +91,16 @@ contract('DividendPayableToken Complex Dividend', function (accounts) {
 
       user2OwnsAfter.should.be.bignumber.below(user2Owns.plus(expectedDividendForAccount2).plus(1000));
       user2OwnsAfter.should.be.bignumber.above(user2Owns.plus(expectedDividendForAccount2).minus(1000));
-  
+
+    });
+
+    it('should not allow starting a new dividend period before the 90 days cooldown', async function () {
+
+      await data.token.startNewDividendPeriod();
+      var increaseTime = (await latestTime()) + duration.days(89);
+      await increaseTimeTo(increaseTime);
+      assertRevert(data.token.startNewDividendPeriod());
```

Comment:
and this `assertRevert(data.token.startNewDividendPeriod());` with 
`await data.token.startNewDividendPeriod().should.be.rejectedWith(EVMRevert);`

because it breaks other tests in file

Reply:
Sure, no problem with that. 
BTW, don't you want to follow this pattern in other files, as I was reusing the code form the: DividendPayableTokenBasic.test.js file.

Reply:
We try to fallow `rejectedWith(EVMRevert)` but couple of test was written some time ago so we do not changed it.